ThunderCloud/QuizGridCell.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  QuizGridCell.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 16/02/2018.
+//  Copyright © 2018 threesidedcube. All rights reserved.
+//
+
+import UIKit
+
+/// Legacy equivalent of `QuizGridItem`
+class QuizGridCell: QuizGridItem {
+
```

Comment:
Move up the brackets to be on one line if it's just a dummy implementation like this